backend/routes/performance_routes.py

```diff
@@ -52,3 +52,19 @@ def update_performance(performance_id):
 def get_adjudications(performance_id):
 	adjudications = Adjudication.get_by(performance_id=performance_id)
 	return jsonify({adjudication.id: adjudication.to_dict() for adjudication in adjudications})
+
+@blueprint.route('/<performance_id>/adjudications', methods=['POST'])
+def create_adjudication(performance_id):
+    adjudication_json = request.get_json()
+    adjudication_json['performance_id'] = performance_id
+    new_adjudication = Adjudication.create(**adjudication_json)
+
+    return jsonify(new_adjudication.to_dict())
+
+@blueprint.route('/<performance_id>/adjudications/<adjudications_id>', methods=['POST'])
+def update_adjudication(performance_id, adjudications_id):
+	adjudication = Adjudication.get(adjudications_id)
+	adjudication_json = request.get_json()
+	adjudication.update(**adjudication_json)
```

Comment:
I think imran mentioned that this doesn't update the adjudication local var in place, so
```
adjudication = adjudication.update(...)
```